edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -964,29 +915,30 @@ If you decide to modify the details of a competition, including the advancement
 Running behind schedule is the most common schedule deviation at WCA competitions. You and the organization team have a number of tools at your disposal to potentially get the competition back on track:
 
 - Continuing the competition as-is and ending later than expected
-- Reducing lunch time
-- Reducing advancement conditions, cutoffs, or time limits
+- Reducing lunchtime
```

Comment:
Maybe we should be more specific with this?  Reducing lunchtime might indicate something other than the duration.  Maybe "Reducing the allotted time for lunch"?

Reply:
Yes, I like that